src/main/java/com/amazon/opendistroforelasticsearch/sql/domain/Condition.java

```diff
@@ -242,7 +243,7 @@ public Condition(CONN conn, String name, SQLExpr nameExpr, String oper,
         if (OPEAR.operStringToOpear.containsKey(oper)) {
             this.opear = OPEAR.operStringToOpear.get(oper);
         } else {
-            throw new SqlParseException(oper + " is not a supported operation");
+            throw new SqlParseException("Unsupported operation: " + oper);
```

Comment:
Maybe better to use `SemanticAnalysisException` for consistency here and elsewhere.

Reply:
Done now, thanks!